user_modules/shared_testcases/testCasesForPolicyTable.lua

```diff
@@ -927,9 +926,12 @@ function testCasesForPolicyTable:flow_SUCCEESS_EXTERNAL_PROPRIETARY(self, app_id
   EXPECT_HMIRESPONSE(RequestId_GetUrls,{result = {code = 0, method = "SDL.GetURLS", urls = endpoints} } )
   :Do(function(_,_)
     self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
-    { requestType = "PROPRIETARY", fileName = ptu_file_name})
+    { requestType = "PROPRIETARY", fileName = SystemFilesPath .. pts_file_name})
     EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY"})
-    :Do(function()
+    :Do(function(_, d2)
+      if not (d2.binaryData ~= nil and string.len(d2.binaryData) > 0) then
+        self:FailTestCase("PTS was not sent to Mobile in payload of OnSystemRequest")
```

Comment:
@dboltovskyi, this function is used in many scripts for successful PTU. In case execution enters in this if, PTU will not be performed, and the script will fail. I think that this check should be done in specific scripts that are checking this notification.
If you want to leave this check, will be better just to print error, without exit of PTU flow.